data/json/npcs/talk_tags.json

```diff
@@ -537,5 +537,81 @@
             "I'm getting my ass out of here!  You'd better do the same, <name_g>!",
             "Fire in the hole, motherfuckers!"
         ]
+    },{
+        "type": "snippet",
+        "category": "<shitty>",
+        "text": [
+            "shitty",
+            "lame",
+            "crappy",
+            "unpleasant",
+            "<swear> <shitty>",
+            "<very> <shitty>",
+            "awful",
+            "terrible",
+            "horrible",
+            "miserable",
+            "fucked-up"
+        ]
+    },{
+        "type": "snippet",
+        "category": "<dumb>",
+            "text": [ 
+            "stupid", 
+            "idiotic", 
+            "dumb", 
+            "dumb-ass", 
+            "moronic", 
+            "mickey mouse", 
+            "shit-for-brains" 
+        ]
+    },{
+        "type": "snippet",
+        "category": "<zombie>",
+        "text": [ 
+            "zombie", 
+            "Z", 
+            "shambler", 
+            "goo-puker", 
+            "walker", 
+            "walking corpse", 
+            "<swear> <zombie>", 
+            "living dead" 
+        ]
+    },{
+        "type": "snippet",
+        "category": "<zombies>",
+        "text": [ 
+            "zombies", 
+            "Z", 
+            "dead", 
+            "<swear> zombies", 
```

Comment:
Do you mean <swear> <zombies>?

Reply:
Didn't mean it, but no reason for it to not be that.

Reply:
In zombies there is &lt;swear&gt; &lt;zombie&gt;, so I figured it was following that pattern and you just forgot to add the &lt;&gt;.

Reply:
I think at the time, I was thinking that I'd like 'zombies' to come up more often, but that was when I thought I'd have more esoteric weird terms in the snippet. Most of these are terms people would use often, so putting the second term as a snippet works just fine